config/prod.exs

```diff
@@ -1,65 +1,29 @@
 use Mix.Config
 
-# For production, we configure the host to read the PORT
-# from the system environment. Therefore, you will need
-# to set PORT=80 before running your server.
-#
-# You should also configure the url host to something
-# meaningful, we use this information when generating URLs.
-#
-# Finally, we also include the path to a manifest
-# containing the digested version of static files. This
-# manifest is generated by the mix phoenix.digest task
-# which you typically run after static files are built.
 config :espi_dni, EspiDni.Endpoint,
   http: [port: {:system, "PORT"}],
-  url: [host: "example.com", port: 80],
-  cache_static_manifest: "priv/static/manifest.json"
+  url: [scheme: "https", host: "espi-dni.herokuapp.com", port: 443],
```

Comment:
Should the host better be a "HOST" env variable to allow deployment to different apps?